dump/match_infra.py

```diff
@@ -0,0 +1,287 @@
+import re, json, os, sys
+from dump.helper import verbose_print
+from swsscommon.swsscommon import SonicV2Connector, SonicDBConfig
+
+error_dict  = {
```

Comment:
General comment: This looks more like C-style approach with error code enumerations.
Can we use exception throw/catch approach for error handling? I see in a lot of places the error handling can be really simplified with this approach.

Reply:
This class was designed to return, a dictionary, with "error" as one of the key (if founf), no exceptions are raised from the MatchEngine. Hence choose to proceed with this approach.